sdks/python/apache_beam/coders/standard_coders.yaml

```diff
@@ -0,0 +1,39 @@
+coder:
+  urn: "beam:coders:bytes:0.1"
+nested: false
+examples:
+  "abc": abc
```

Comment:
should the format force listing nested and unnested examples as part of the same coder test definition instead of having two separate coder test definitions?